RubeesEat/wwwroot/css/BillDetails.css

```diff
@@ -0,0 +1,59 @@
+div.bill-summary {
+    padding: 1.5rem;
+    background-color: var(--secondary-color);
+    border: 1px solid var(--primary-color);
+    border-radius: 8px;
+    max-width: 400px;
+    margin: 1rem auto;
+    text-align: center;
+}
+
+.billDetails{
```

Comment:
I think we should get rid of the hover effect, because this indicates a button which is actually not the case